explorer/src/Pos/Explorer/Web/Api.hs

```diff
@@ -105,22 +125,22 @@ type GenesisPagesTotal = API
     :> "total"
     :> QueryParam "pageSize" Word
     :> QueryParam "filter" CAddressesFilter
-    :> Get '[JSON] (Either ExplorerError PageNumber)
+    :> ExRes Get PageNumber
 
 type GenesisAddressInfo = API
     :> "genesis"
     :> "address"
     :> QueryParam "page" Word
     :> QueryParam "pageSize" Word
     :> QueryParam "filter" CAddressesFilter
```

Comment:
You can use `DQueryParam` from `Pos.Util.Servant` here. It will take `def` value if nothing is specified.

Reply:
Oh, looks like there is no such function yet. But you can add one here:
https://github.com/input-output-hk/cardano-sl/blob/master/lib/src/Pos/Util/Servant.hs#L512